app/components/support_interface/permissions_list_review_component.html.erb

```diff
@@ -11,6 +11,14 @@
     <li><%= render IconComponent.new(type: 'cross') %> Manage users</li>
   <% end %>
 
+  <% if FeatureFlag.active?(:interview_permissions) %>
+    <% if permissions['set_up_interviews'] %>
```

Comment:
Maybe not for this PR as I understand the addition here is consistent with the rest of the file, but we could DRY this up to something like:

```erb
<li><%= render IconComponent.new(type: (permissions['set_up_interviews'] ? 'check' : 'cross')) %> Set up interviews</li>
```